src/uni_modules/wot-ui/components/wd-calendar-view/monthPanel/month-panel.vue

```
@@ -389,8 +389,16 @@ async function scrollIntoView() {
     activeDate = props.value
   }
 
-  if (!activeDate) {
-    activeDate = Date.now()
+  // 当传入日期为空或超出可选范围，修改为当前日期
+  if (!activeDate || activeDate < props.minDate || activeDate > props.maxDate) {
+		activeDate = Date.now()
```

ESLint

[error] Replace `↹↹` with `····`

(prettier/prettier)